packages/twenty-ui/src/theme/provider/CSSModuleConstants.css

```diff
@@ -0,0 +1,67 @@
+:root {
+  --twentycrm-spacing-multiplicator: 4;
+  --twentycrm-border-radius-sm: 4px;
+  --twentycrm-font-weight-medium: 500;
+
+  /* Grays */
+  --twentycrm-gray-100: #000000;
```

Comment:
any chance we don't have to re-declare the theme twice? We should at least have one unique source of truth (and then maybe generate constant for css-modules and emotion theme from there)